exammode/doc/EXAM.md

```diff
@@ -0,0 +1,210 @@
+Electrical Exam Documentation
+=============================
+
+This documentation describes the functionalities added to the A+ system
+for electrical examination during Summer 2019 by a few summer workers.
+
+Table of contents
+-----------------
+
+* [Exammode app](#exammode-app)
+* [Exam Management Page](#exam-management-page)
+* [Selection of Exam Templates](#selection-of-exam-templates)
+
+Exammode app
+------------
+
+The exammode app is the bread and butter of the designed electrical exam system. 
+
+An exam session is the basic unit for the system and all  features of the system are linked to one session and can be accessed by viewing that session. These features are
+
+For students:
+
+* See exam dates and locations
+* Request translation of  exam
+* Request accelerated grading and reporting of results
```

Comment:
Just checking, are these features really implemented? Can students request accelerated grading for their exam etc.?

Reply:
Marked them as a feature ideas not yet implemented,